### DISCUSSION TITLE
How-To Get Tabby to show Tabby Home screen by default

### DISCUSSION THREAD

**Discussion by atholcomb** [Eugeny/tabby#9728] [Q&A]
I really enjoy using Tabby and have found that it suits all my needs. There is **one** thing that I'd like to see that it could do. Specifically, is there a way that Tabby can (on startup) open **just** the recent connections screen without opening _any_ shell whatsoever?? 

See picture attached.
![tabby_home](https://github.com/Eugeny/tabby/assets/13822733/bfd0ed30-6964-47ae-a60c-bfd734bbb117)

**Comment by Eugeny** [comment]
Yes, disable tab recovery and an empty default tab in Settings -> Terminal -> Startup

**Comment by atholcomb** [comment]
PERFECT! Thank you for the wonderful app!
